wikum/private_config.py

```diff
@@ -1,11 +1,11 @@
 MYSQL_PROD = {
```

Comment:
i'm guessing you do not want to commit changes to this file? :)

Reply:
Oh yes, thanks for catching that! I've reverted this file now.